src/bptl/work_units/brp/tasks.py

```diff
@@ -37,3 +38,49 @@ def perform(self) -> dict:
         is_above_age = None if leeftijd is None else leeftijd >= age
 
         return {"isAboveAge": is_above_age}
+
+
+@register
+class DegreeOfKinship(WorkUnit):
+    """
+    Fetches BRP API and returns flag if certain two people have a specified degree of kinship
+
+    Required process variables:
+
+    * ``burgerservicenummer1``: BSN of the first person
+    * ``burgerservicenummer2``: BSN of the second person
+
+    The task sets the process variables:
+
+    * ``kinship``: integer, which represents the degree of kinship (blood relations). Values can be in range [1..4]
+    """
+
+    def perform(self) -> dict:
+        variables = self.task.get_variables()
+        bsn1 = variables["burgerservicenummer1"]
+        bsn2 = variables["burgerservicenummer2"]
+
+        if bsn1 == bsn2:
+            return {"kinship": None}
+
+        client = get_client_class()()
```

Comment:
Is this a special client that I don't know about but talks json+hal? :)

Reply:
We'll do some cleanup w/r to the client class stuff later, especially with the pendin zgw-consumers rework.